app/src/main/java/com/example/blah/mobilestudio/FileViewer/FileFragment.java

```diff
@@ -132,8 +130,8 @@ protected String doInBackground(Void... params) {
                 br.close();
                 String result = stringBuilder.toString();
                 result = highlightSourceCode(result);
```

Comment:
Considering the next 40 lines or so, it seems like they don't really fit in with the concerns of the FileFragment. There's no logic specific to the UI being done here, just changes made to a string.
 I think perhaps we should create a new class called TextFormatter - and move the highlightSource code method to that class, along with the replaceAll() method call.